lib/state/simperium/middleware.ts

```diff
@@ -223,6 +223,12 @@ export const initSimperium = (
       });
     }
   });
+  preferencesBucket.channel.once('ready', () => {
+    dispatch({
+      type: 'REMOTE_ANALYTICS_UPDATE',
+      allowAnalytics: getState().data.analyticsAllowed,
```

Comment:
I think here we want `allowAnalytics: !!(await preferencesBucket.get('preferences-key'))?.analytics_enabled` or whatever it takes to do that. maybe this works too and it won't matter much.

Reply:
Resolved